tests/postgres_tests/fields.py

```diff
@@ -0,0 +1,31 @@
+"""
+Indirection layer for PostgreSQL-specific fields, to not fail when tests run
+with another backend than PostgreSQL.
+"""
+from django.db import models
+
+try:
+    from django.contrib.postgres.fields import (
+        ArrayField, BigIntegerRangeField, DateRangeField, DateTimeRangeField,
+        FloatRangeField, HStoreField, IntegerRangeField,
+    )
+except ImportError:
+    class DummyArrayField(models.Field):
```

Comment:
is this necessary to even allow the import of the migration file, or could it also just be a mock (like how the extension operations are handled)?